components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/LocalAuthenticatorConfig.java

```diff
@@ -82,6 +86,9 @@ public static LocalAuthenticatorConfig build(OMElement localAuthenticatorConfigO
                 if (member.getText() != null && member.getText().trim().length() > 0) {
                     localAuthenticatorConfig.setEnabled(Boolean.parseBoolean(member.getText()));
                 }
+            } else if ("Tags".equals(member.getLocalName())) {
```

Comment:
Can't we use the TAG constant that is introduced https://github.com/wso2/carbon-identity-framework/pull/3570/files#diff-9461270f09d52c105a65affa1fdc9fbd740274afc25204ffd5ec2bf49e6975b9R133 in this place instead of the string "Tags"?

Reply:
Address with https://github.com/wso2/carbon-identity-framework/pull/3779 PR